packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -50,6 +50,8 @@ import { autoUpdate, flip, hide, useFloating } from '@floating-ui/react';
 import { TranslateWithId } from '../../types/common';
 import { useFeatureFlag } from '../FeatureFlags';
 
+// TODO: Is there supposed to be handling for `InputKeyDownEscape` in this
+// component?
```

Comment:
I don't think so? Escape is handled here:

https://github.com/carbon-design-system/carbon/blob/381edb24bc38a243f0a1997da0274c85bff68ab8/packages/react/src/components/ComboBox/ComboBox.tsx#L1023-L1029

Usually that means the default downshift behavior isn't what we want and so it's handled on keydown before downshift gets a chance to run it's callbacks/behavior. You could look into it though, it might be possible to reduce or simplify the code by handling it in the state manager.

Reply:
Noted. The reason I asked is that the Escape behavior doesn't match the accessibility documentation.